webapp: Adding support for MinTls, AlwaysOn, FtpsState properties for an app & slot (#10867)
Azure/azure-powershell: src/Websites/Websites/ChangeLog.md

        - Additional information about change #1
-->
## Upcoming Release
* Set-AzWebapp and Set-AzWebappSlot supports AlwaysOn, MinTls and FtpsState properties

> * Fixing issue where etting HttpsOnly along with changing AppservicePlan at the same time using the single Set-AzWebApp Command, was resetting HttpsOnly to default value

Reviewer: ```suggestion
* Fixing issue where editing HttpsOnly along with changing AppServicePlan at the same time using the single Set-AzWebApp Command, will reset HttpsOnly to default value
```

Author: oops will fix the typo as well
